src/app/services/communibee-backend/volunteering-events/volunteering-event.ts

```diff
@@ -0,0 +1,33 @@
+import BackendModel from '../backend-model';
+
+export interface VolunteeringRequestOfferBase {
+  numberOfVolunteers: number;
+  contact: {
+    name: string;
+    phone: string;
+    email: string;
+  };
+  dateRange: {
+    from: Date;
+    to: Date;
+  };
+  availableWeekdays: number[];
+  timeRange: {
+    from: Date;
+    to: Date;
+  };
+  notes: string[];
+}
+
+export interface VoulunteeringEvent {
+  request: VolunteeringRequestOfferBase;
+  offer: VolunteeringRequestOfferBase;
+  date: Date;
+  userInitiated: {
```

Comment:
We've changed this type:
https://github.com/CommuniBee/CommuniBee-Back-End/pull/19/files#diff-fe29882891aa86240208de2479d82da5